schemas/v1.0/.openpublishing.publish.config.schema.json

```diff
@@ -0,0 +1,144 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Do we want to mark certain properties as required?

Reply:
For op.config.json yes